stable/redis-ha/templates/redis-server-statefulset.yaml

```diff
@@ -8,17 +23,20 @@ metadata:
     redis-node: "true"
 {{ include "labels.standard" . | indent 4 }}
 spec:
+  serviceName: {{ template "redis-ha.fullname" . }}-server-svc
   replicas: {{ .Values.replicas.servers }}
+  updateStrategy:
+    type: OnDelete
   template:
     metadata:
       labels:
-        app: {{ template "redis-ha.name" . }}
-        release: {{ .Release.Name }}
-        component: server
         name: {{ template "redis-ha.fullname" . }}-server
         redis-node: "true"
+{{ include "labels.standard" . | indent 8 }}
     spec:
-      serviceAccountName: {{ template "redis-ha.serviceAccountName" . }}
+      {{ if .Values.rbac.create }}
+      serviceAccountName: {{ template "redis-ha.fullname" . }}-svcacct
```

Comment:
Does this work for you with the values as given in values.yaml? As when I try to deploy this, the ServiceAccount that's created gets the name `idolized-lemur-redis-ha`, without the `-svcacct` added. When I look at the definition in _helpers.tpl, it seems that the template "redis-ha.serviceAccountName" does not have this `svcacct` part.

Reply:
You are right. I think this was a rebase mistake